first-party/jib-layer-filter-extension-gradle/README.md

```diff
@@ -0,0 +1,47 @@
+# Jib Layer-Filter Extension
+
+A general-purpose layer-filter extension that enables fine-grained layer control, including deleting files and moving files into new layers. Note that the extension can only filters files put by Jib; it cannot filter files in the base image.
+
+## Examples
+
+```grdle
```

Comment:
`grdle` -> `gradle`

also should we include how to include the actual jar into the build?

Reply:
That's a good idea. Also updated the Ownership extension docs.